### PR TITLE
Update BlazingPizza app to 5.0

### PR DESCRIPTION
**Pull request by mkArtak**

### CODE REVIEW

**Comment by jongalloway** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet-presentations/blazor-workshop?pullRequest=292) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-presentations/blazor-workshop?pullRequest=292) before we can accept your contribution.<br/><br/>:x: mkArtak [sign now](https://cla.dotnetfoundation.org/dotnet-presentations/blazor-workshop?pullRequest=292)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-presentations/blazor-workshop?pullRequest=292) it.</sub>

**Comment by mkArtak** [comment]
@danroth27, @SteveSandersonMS updated this to 5.0-rc2. I guess when GA ships, we'll just update to GA then mainly by updating package versions. I've manually validated the app - things work.

**Comment by danroth27** [comment]
We need to do a full pass through all of the labs to see where we should be using new features from .NET 5 and update all of the save points.

**Comment by KenSpr** [comment]
Thanks all for the amazing great work!

**Comment by mkArtak** [comment]
I've now upgraded all step projects to 5.0 and everything works as is.

@SteveSandersonMS will you be able to review the current state and see what 5.0 specific changes we should integrate to better showcase new features.

**Review by danroth27 — APPROVED** [review]
LGTM. We should still at some point go through the workshop and look for places to take advantage of new .NET 5 features (CSS isolation, etc), but I think it's good to merge as is.

File: Directory.Build.props [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <Project>
     <PropertyGroup>
-        <AspNetCoreVersion>5.0.0-preview.7.20326.3</AspNetCoreVersion>
-        <BlazorVersion>5.0.0-preview.7.20326.3</BlazorVersion>
-        <EntityFrameworkVersion>5.0.0-preview.7.20326.1</EntityFrameworkVersion>
-        <SystemNetHttpJsonVersion>5.0.0-preview.7.20326.1</SystemNetHttpJsonVersion>
+        <AspNetCoreVersion>5.0.0.0</AspNetCoreVersion>
```
Comment (danroth27) [inline Directory.Build.props:3]: Shouldn't these just be 5.0.0? I don't think we ever use a fourth digit in our package version numbers.

File: src/BlazingPizza.Client/Program.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ public class Program
         public static async Task Main(string[] args)
         {
             var builder = WebAssemblyHostBuilder.CreateDefault(args);
-            builder.RootComponents.Add<App>("app");
+            builder.RootComponents.Add<App>("#app");
```
Comment (danroth27) [inline src/BlazingPizza.Client/Program.cs:15]: We should make these same changes in the save points.

File: src/BlazingPizza.Client/wwwroot/index.html [hunk 3]
```diff
@@ -12,9 +12,7 @@
     <title>Blazing Pizza</title>
 </head>
 <body>
-    <app>
-        <div class="loading-bar"></div>
-    </app>
+    <div id="app"><div class="loading-bar"></div></div>
```
Comment (danroth27) [inline src/BlazingPizza.Client/wwwroot/index.html:15]: We should make these same changes in the save points.

File: src/BlazingPizza.Server/Startup.cs [hunk 4]
```diff
@@ -18,9 +18,9 @@ public Startup(IConfiguration configuration)
         public IConfiguration Configuration { get; }
 
         public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddMvc()
-                .AddNewtonsoftJson();
+        {
```
Comment (danroth27) [inline src/BlazingPizza.Server/Startup.cs:21]: We should make these same changes in the save points.

**Pull request merged**